frontend/src/components/files/FilesTableFileEntry.tsx

```diff
@@ -27,7 +27,7 @@ export function FilesTableFileEntry(props: FilesTableFileEntryProps) {
 			try {
 				let url = "";
 				if (file.thumbnail_id) {
-					url = await generateVisDataDownloadUrl(file.thumbnail_id);
+					url = await downloadThumbnail(file.thumbnail_id);
```

Comment:
Since now we enabled cookie authentication, we might be able to just say
```
<img src={`${config.hostname}/api/v2/thumbnails/${file.thumbnailId}`} alt="thumbnail" width="24" />
```